namer/dtab.go

```diff
@@ -57,6 +57,9 @@ func (dtab Dtab) String() string {
 func (dtab Dtab) Pretty() string {
 	maxPfxLen := 0
 	for _, d := range dtab {
+		if d == nil {
+			continue
+		}
```

Comment:
This seems fine to me, but for clarity, were you actually running into situations where the `dtab` slice contained nil values, or is this just to be extra cautious?

Reply:
Ok, great, thanks for clarifying and filing that issue.